src/main/resources/archetype-resources/src/main/resources/atlassian-plugin.xml

```diff
@@ -7,4 +7,33 @@
 
     <resource type="i18n" name="i18n" location="i18n/plugin"/>
 
-</atlassian-plugin>
\ No newline at end of file
+#if( $generateFrontend == "true" )
+    <resource type="download" name="js/" location="/js"/>
+
+    <!-- .................................................................. -->
+    <!-- TOP LEVEL APP MENU -->
+    <!-- .................................................................. -->
+    <web-item key="app-item" section="system.top.navigation.bar">
```

Comment:
**system.top.navigation.bar** is only defined in jira, in confluence this is **system.header/left**